bin/add-practice-exercise

```diff
@@ -66,7 +66,7 @@ fi
 
 ./bin/fetch-configlet
 ./bin/configlet create --practice-exercise "${slug}" --author "${author}" --difficulty "${difficulty}"
-jq '.exercises.practice |= sort_by(.difficulty, .name)' config.json > config.json.tmp && mv config.json.tmp config.json
+jq '.exercises.practice |= sort_by(.difficulty, (.name|ascii_upcase))' config.json > config.json.tmp && mv config.json.tmp config.json
 bin/fetch-configlet && bin/configlet fmt --update --yes
```

Comment:
Maybe as a follow-up PR: replace these two lines with a call to `reorder-exercises`?